app/controllers/jobseekers/job_applications_controller.rb

```diff
@@ -1,19 +1,30 @@
 class Jobseekers::JobApplicationsController < Jobseekers::BaseController
-  helper_method :job_application, :vacancy
+  helper_method :job_application, :review_form, :vacancy
 
   def new
     request_event.trigger(:vacancy_apply_clicked, vacancy_id: vacancy.id)
   end
 
   def create
     new_job_application = current_jobseeker.job_applications.find_or_initialize_by(vacancy: vacancy)
-    new_job_application.draft!
-    redirect_to jobseekers_job_application_build_path(new_job_application, :personal_details)
+
+    if new_job_application.persisted?
+      redirect_to jobseeker_root_path, notice: "You already applied for this job"
```

Comment:
Where do these pieces of content come from? I believe this one should be 'You have already'.

Reply:
First name: Connor, second name: Placeholder

Reply:
I know there is a ticket coming about this so I wasn't too concerned about the content (or even where it should redirect) and that's why I didn't add a translation either. If you prefer I can just omit this bit of code completely

Reply:
Can you set it to `notice: "You have already applied for this job TODO: confirm content is accurate"`

Reply:
Actually - in hindsight I'm going to leave this part completely for https://dfedigital.atlassian.net/browse/TEVA-2036